docs/user-guide/building-isos.rst

```diff
@@ -18,7 +18,16 @@ wish to boot with your Cobbler installation: ``grob2-ARCH-efi-modules``.
 The script mkgrub.sh
 ####################
 
-This script can create a bootable grub2 Bootloader in the EFI format.
+This script can create a bootable grub2 Bootloader in the EFI format. Thus it collects all modules and creates a
+bootable grub2 bootloader. The folder where this is executed is not relevant.
+
+To build grub bootloaders for other architectures install the packages and then execute the script against the newly
+installed directories. openSUSE has enabled you to do this but other distros may not decide to do this. If your distro
```

Comment:
openSUSE has enabled this for you but... . If your distro does not enable this for you, you need to enable this yourself.